hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/table/functional/TestHoodieSparkRollback.java

```diff
@@ -111,14 +113,19 @@ protected HoodieWriteConfig getConfigToTestMDTRollbacks(Boolean autoCommit, Bool
         .withRollbackUsingMarkers(true)
         .withMetadataConfig(HoodieMetadataConfig.newBuilder().enable(mdtEnable).build())
         .build();
+    cfg.setValue(HoodieWriteConfig.WRITE_TABLE_VERSION, "6");
+    return cfg;
   }
 
   /**
    * Scenario: data table is updated, no changes to MDT
    */
   protected void testRollbackWithFailurePreMDT(HoodieTableType tableType) throws Exception {
     initBasePath();
-    HoodieTableMetaClient metaClient = getHoodieMetaClient(tableType);
+    TypedProperties props = new TypedProperties();
```

Comment:
can we add a validation at the end of every test to confirm that table config -> table version is 6.

Reply:
Addressed